languages/en/timers.genie

```diff
@@ -372,19 +378,38 @@ timer = {
 
         !turking {
             optional_once each interval:constant_Measure_ms => new Ast.Stream.Timer(Ast.Value.Date.now(), interval, null, TIMER_SCHEMA);
-            once_per_day => new Ast.Stream.AtTimer([TIME_MORNING], null, AT_TIMER_SCHEMA);
+            once_per_day => new Ast.Stream.Timer(Ast.Value.Date.now(), new Ast.Value.Measure(1, 'day'), null, TIMER_SCHEMA);
+            // twice_per_day and thrice_per_day still need to be AtTimer until we can explicitly set the frequency param in Timer
             twice_per_day => new Ast.Stream.AtTimer([TIME_MORNING, TIME_EVENING], null, AT_TIMER_SCHEMA);
             thrice_per_day => new Ast.Stream.AtTimer([TIME_MORNING, TIME_NOON, TIME_EVENING], null, AT_TIMER_SCHEMA);
```

Comment:
These two (twice_per_day / thrice_per_day) need to go

Reply:
We wanted to change them to Timer but couldn't set the freq parameter explicitly. The main problem is that the words 'twice' and 'thrice' wouldn't get mapped if we remove these.